adapters/polychain-monsters/index.js

```diff
@@ -0,0 +1,115 @@
+        this.contract = "0x85f0e02cb992aa1f9f47112f815f519ef1a59e2d";
+        this.events = ["Transfer"];
+        this.pathToAbi = path.join(__dirname, "./abi.json");
+        this.range = 500;
+        this.chunkSize = 6;
+        this.sdk = new Binance(this);
+    }
+
+    run = async () => {
+        const s = await this.getSymbol();
+        this.sdk = this.loadSdk();
+        this.symbol = s;
+        await this.sdk.run();
+    };
+
+    loadSdk = () => {
+        return new Binance(this);
+    };
+    getSymbol = async () => {
+        const resp = await axios.get(
+            `${URL}/token-metadata?key=${KEY}&token_address=${this.token}&protocol=${this.protocol}`,
+            {
+                headers: {
+                    "User-Agent":
+                        "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4619.141 Safari/537.36",
+                },
+            },
+        );
+        const symbol = resp.data;
+        return symbol;
+    };
+    getPrice = async timestamp => {
+        const resp = await axios.get(
+            `${URL}/token-price?key=${KEY}&token_address=${this.token}&protocol=${this.protocol}&timestamp=${timestamp}`,
+            {
+                headers: {
+                    "User-Agent":
+                        "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4619.141 Safari/537.36",
+                },
+            },
+        );
+        return resp.data;
+    };
+    stop = async () => {
+        this.sdk.stop();
+    };
+
+    getBuyer = async event => {
+        const buyer = event.returnValues.owner;
+        if (event.event === "Transfer") {
+            const txReceipt = await this.sdk.getTransactionReceipt(event.transactionHash);
+            if (txReceipt === null) {
+                return null;
+            }
+            return txReceipt.from;
+        }
+        return buyer;
+    };
+
+    process = async event => {
+        const block = await this.sdk.getBlock(event.blockNumber);
+        const timestamp = moment.unix(block.timestamp).utc();
+        const po = await this.getPrice(block.timestamp);
+        const nativePrice = new BigNumber(event.returnValues.cost).dividedBy(10 ** this.symbol.decimals);
+        const buyer = await this.getBuyer(event);
+        if (!buyer) {
+            return;
+        }
+
+        const labelHash = event.returnValues.label;
+        const tokenId = new BigNumber(labelHash, 16).toFixed();
+        const entity = {
+            provider_name: this.name, // the name of the folder
+            provider_contract: this.contract, // the providers contract from which you get data
+            protocol: this.protocol,
+            nft_contract: this.contract,
+            nft_id: tokenId,
+            token: this.token,
+            token_symbol: this.symbol.symbol,
+            amount: 1,
+            price: nativePrice.toNumber(),
```

Comment:
Does not pass tests, returns an entity without price
`Got an error in tester Error: Missing entity key: price`
Make sure to pass the tests before committing